src/test/java/LoginTests.java

```diff
@@ -8,18 +10,42 @@
 
 public class LoginTests extends BaseTest {
     @Test
-    public void LoginEmptyEmailPasswordTest() {
+    public void successfulLoginTest() {
 
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
 
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+//        Find email field inside the web page
+        WebElement emailInput = driver.findElement(By.cssSelector("[type='email']"));
+        emailInput.click();
+        emailInput.clear();
+        emailInput.sendKeys("demo@class.com");
 
-        String url = "https://qa.koel.app/";
+//        Put password in password field
+        WebElement passwordInput = driver.findElement(By.cssSelector("[type='password']"));
+        passwordInput.click();
+        passwordInput.clear();
+        passwordInput.sendKeys("te$t$tudent");
+
+//        Click submit
+        WebElement submitLogin = driver.findElement(By.cssSelector("button[type='submit']"));
+        submitLogin.click();
+
+//        Check if avatar is displayed
+        WebElement avatar = driver.findElement(By.cssSelector(".avatar"));
+        Assert.assertTrue(avatar.isDisplayed());
+
+
+
+
+
+
+
+
+<<<<<<< Updated upstream
+        String url = "https://bbb.testpro.io/";
         driver.get(url);
         Assert.assertEquals(driver.getCurrentUrl(), url);
         driver.quit();
+=======
+>>>>>>> Stashed changes
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/356a4576-74ec-4fe8-b7ae-3a6ee61c3e7e)

HINT

Conflict in the code, indicated by the comment <<<<<<< Updated upstream. This conflict usually occurs when two or more versions of the code are being merged together, and there are conflicting changes in the same section of code.

To resolve the conflict, you need to manually edit the code and choose which version to keep or merge the changes from both versions. Here's an example of how you can resolve the conflict:

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/d6796e49-f25d-4c9f-b72a-893668c47896)

Resolving a merge conflict on GitHub https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github